src/Symfony/Bundle/FrameworkBundle/Templating/Helper/CodeHelper.php

```diff
@@ -16,22 +16,29 @@
 
 /**
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @internal
  */
 class CodeHelper extends Helper
 {
     protected $fileLinkFormat;
+    /**
+     * @deprecated since Symfony 4.2
+     */
     protected $rootDir;
+    protected $projectDir;
```

Comment:
should the new property be private instead ? I don't think it needs to be an extension point

Reply:
of course, done